README.md

```diff
@@ -281,9 +281,13 @@ And finally, you are ready to go! Or at least you should be...
 
 **Iris Mestres** | [Contact me][deliaBlue]
 
-### Junior Contributors
+**T**
 
+### Junior Contributors
 
+**Tassnime Reddahi: Let me infect you with some of my positive eneregy**
```

Comment:
The format should be:
"**Tassnime Reddahi** | [Tassnimered][Tassnimered] | Let me infect you with some of my positive eneregy"